lib/strava/api/client.rb

```diff
@@ -15,6 +15,8 @@ class Client < Strava::Web::Client
 
       attr_accessor(*Config::ATTRIBUTES)
 
+      attr_accessor :ratelimit_status
```

Comment:
I added the RatelimitStatus class to the client, making it possible to check the limits after each call. Added a boolean flag, too, as not every call gets rate limit in its headers.